wavefront_sdk/common/__init__.py

```diff
@@ -9,9 +9,11 @@
 
 # flake8: noqa
 
+from wavefront_sdk.common.application_tags import ApplicationTags
+from wavefront_sdk.common.heartbeater_service import HeartbeaterService
 from wavefront_sdk.common.utils import metric_to_line_data
 from wavefront_sdk.common.utils import histogram_to_line_data
 from wavefront_sdk.common.utils import tracing_span_to_line_data
 
-__all__ = ['metric_to_line_data', 'histogram_to_line_data',
-           'tracing_span_to_line_data']
+__all__ = ['ApplicationTags', 'HeartbeaterService', 'metric_to_line_data',
```

Comment:
You're not hiding anything here, so overriding `__all__` doesn't seem to be necessary here